README.md

```diff
@@ -55,6 +55,33 @@ You'll need [Clang 3.7](http://llvm.org/releases/download.html#3.7.0).
 You can then pass the contents of `prog.o` to `ubpf_load_elf`, or to the stdin of
 the `vm/test` binary.
 
+## Benchmarking and Instruction Counts
+
+Using the ```-i``` option instructs the test program to count the
```

Comment:
Nit: I think you only want one backtick here.  Three backticks are usually used for code blocks.